sdk/identity/identity/review/identity.api.md

```diff
@@ -185,6 +219,26 @@ export class ManagedIdentityCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
     }
 
+// @public
+export abstract class PersistentCredential implements TokenCredential {
```

Comment:
It's too bad we already shipped a bunch of Credential classes. If we had shipped them as interfaces, we could completely hide this base class as an implementation detail.

Reply:
Perhaps we could share implementation in a different way than inheritance.

Reply:
On call with Scott and Brian we agreed on dropping PersistentCredential and InteractiveCredential. Credentials will always only implement TokenCredential.

Reply:
I'll push that in my next commit, so I'll resolve this to help me focus on what's left.